texar/torch/data/tokenizers/bert_tokenizer_utils.py

```diff
@@ -265,13 +265,24 @@ def tokenize(self, text: str) -> List[str]:
                 if cur_substr is None:
                     is_bad = True
                     break
-                sub_tokens.append(cur_substr)
+                sub_tokens_and_span.append((cur_substr, start, end))
                 start = end
 
             if is_bad:
-                output_tokens.append(self.unk_token)
+                output_tokens_and_span.append((self.unk_token, 0, len(chars)))
             else:
-                output_tokens.extend(sub_tokens)
+                output_tokens_and_span.extend(sub_tokens_and_span)
+
+        return output_tokens_and_span
+
+    def tokenize(self, text: str) -> List[str]:
+        """
+        Tokenizes a piece of text into its word pieces, remove span info.
```

Comment:
The term "remove span info" is confusing for users who don't know the internal calls. You should simply describe what this does. You can point to the other function if you would like to tell the users how to preserve the spans.

You should also copy the `longest-match-first algorithm` part here.